### PR TITLE
feat: add email to Auth header when fetching components

### PR DESCRIPTION
**Pull request by yasserfaraazkhan**
- in `.env` file Add a new variable `EMAIL=your.email@example.com`

### CODE REVIEW

File: src/util/jira.ts [hunk 1]
```diff
@@ -6,21 +6,33 @@ const base = 'https://mattermost.atlassian.net';
...
 
   private _auth(r: Request): Request {
     const req = new Request(r);
     if (this.token) {
-      req.headers.set('Authorization', `Bearer ${this.token}`);
+      const encoder = new TextEncoder();
+      const data = encoder.encode(`${this.email}:${this.token}`);
+      const base64 = btoa(String.fromCharCode(...new Uint8Array(data)));
+      const authString = `Basic ${base64}`;
+      req.headers.set('Authorization', authString);
```
Comment (saturninoabril) [inline src/util/jira.ts:35]: nit: I'd request for the `Basic` to go directly when setting up the authorization and not through a separate variable.

```suggestion

      req.headers.set('Authorization', `Basic ${base64}`);
```

**Review by saturninoabril — CHANGES_REQUESTED** [review]
Thanks @yasserfaraazkhan, awesome!

One request and I'm all good.

File: src/util/jira.ts (hunk 1 above)
Comment (yasserfaraazkhan) [reply src/util/jira.ts]: thank you @saturninoabril , I've made the change

**Review by saturninoabril — APPROVED** [review]
Great, thanks Yasser!

**Pull request merged**